src/libsync/putmultifilejob.cpp

```diff
@@ -90,8 +102,10 @@ bool PutMultiFileJob::finished()
     for(const auto &oneDevice : _devices) {
         Q_ASSERT(oneDevice._device);
 
-        if (!oneDevice._device->errorString().isEmpty()) {
-            qCWarning(lcPutMultiFileJob) << "oneDevice has error:" << oneDevice._device->errorString();
+        if (oneDevice._device->isOpen()) {
+            if (!oneDevice._device->errorString().isEmpty()) {
+                qCWarning(lcPutMultiFileJob) << "oneDevice has error:" << oneDevice._device->errorString();
+            }
         }
 
         if (oneDevice._device->isOpen()) {
```

Comment:
Can't we condense this?

```suggestion
        if (oneDevice._device->isOpen()) {
            if (!oneDevice._device->errorString().isEmpty()) {
                qCWarning(lcPutMultiFileJob) << "oneDevice has error:" << oneDevice._device->errorString();
            }
```

Reply:
done